charts/opentelemetry-collector/values.yaml

```diff
@@ -83,6 +83,9 @@ configMap:
   # Specifies an existing ConfigMap to be mounted to the pod
   # The ConfigMap MUST include the collector configuration via a key named 'relay' or the collector will not start.
   existingName: ""
+  # Specifies the relative path to custom ConfigMap template file. This option MUST be used when bundling a custom
+  # ConfigMap template, as it enables pod restart via a template checksum annotation.
```

Comment:
```suggestion
  # Specifies the relative path to custom ConfigMap template file. This option SHOULD be used when bundling a custom
  # ConfigMap template, as it enables pod restart via a template checksum annotation.
```

Reply:
I view this as a SHOULD because you can install the chart without this field set. You'll miss out on the pod restart feature, but it is still possible to use the chart.

Reply:
Good call.